encoding/inputdecoder/decoder_test.go

```diff
@@ -62,13 +62,13 @@ func TestRequestReader(t *testing.T) {
 		reader, err := New(buf)
 		assert.NotNil(t, reader, "expected non-nil reader")
 		assert.NoError(t, err)
-		body, err := reader.NextJSONBytes()
+		body, err := reader.NextYAMLBytes()
```

Comment:
can we change the test input here? The output `{"test":2}` looks more like JSON than YAML here. Would not expect to see `{}` in the YAML bytes.

Reply:
JSON output is actually valid YAML (since JSON is subset of YAML).
This test has JSON requests as input, which uses JSON decoder to parse input and return back JSON bytes (which is valid YAML)

Reply:
TIL! https://en.wikipedia.org/wiki/YAML: 
> ...  so that JSON files are valid YAML 1.2.